### PR TITLE
Added trapz

### LINKED ISSUE #17148: trapz

### PR DESCRIPTION
**Pull request by Bright-in**
Close #17148

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy/functional/frontends/jax/numpy/mathematical_functions.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def absolute(x, /):
     return ivy.abs(x)
 
 
-abs = absolute
```
Comment (YushaArif99) [inline ivy/functional/frontends/jax/numpy/mathematical_functions.py:22]: I think this file was added by mistake. Can you remove it 🙂

File: ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py
Comment (YushaArif99) [inline ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py:1]: this file too doesnt seem relevant to the PR. Can you remove it as well 🙂

File: ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py
Comment (YushaArif99) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py:1]: same goes for this file as well 🙂

File: ivy_tests/test_ivy/test_frontends/test_torch/test_blas_and_lapack_ops.py [hunk 2]
```diff
@@ -675,6 +675,33 @@ def test_torch_pinverse(
...
+    frontend,
+    test_flags,
+):
+    dtype, y, x = dtype_y_x
+    if use_x:
+        test_flags.num_positional_args = 2
+        kwargs = {"y": y, "x": x, "dim": -1}
+    else:
+        test_flags.num_positional_args = 1
+        kwargs = {"y": y, "dx": dx, "dim": dim}
```
Comment (YushaArif99) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_blas_and_lapack_ops.py:696]: I am not sure what you're trying to do here in general 🤔. The frontend helper `helpers.test_frontend_function` takes care of testing the function with a combination of args/kwargs and so there is no need to hardcode this into your test function itself. Any particular reason for adding this check?

Also, the frontend function `trapz` does not contain any argument called `dx` so why are you supplying it here?

**Review by YushaArif99 — CHANGES_REQUESTED** [review]
Hi @Bright-in 

Apologies for the late response. Have been swamped with work recently and so haven't had the time to review any PRs 😅. 

Anyways, could you kindly look into the above-mentioned points? Feel free to discuss in detail any concern you have or to clarify anything I might have misunderstood 🙂

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**